scripts/release-bus-backend-package-strategies.mjs

```
@@ -0,0 +1,104 @@
+    configured.length !== categorized.length ||
+    configured.some((name, index) => name !== categorized[index])
+  )
+    throw new Error(
+      'Every backend deploy unit must have one explicit Release Bus install strategy'
+    );
+}
+
+export function validateReleaseBusBackendLayers(units, layers) {
+  const flattenedLayers = Array.isArray(layers) ? layers.flat() : [];
+  const compareBytes = (left, right) =>
+    Buffer.compare(Buffer.from(left, 'utf8'), Buffer.from(right, 'utf8'));
```

ESLint

[error] 'Buffer' is not defined.

(no-undef)